django/shortcuts.py

```diff
@@ -6,18 +6,25 @@
 from django.http import (
     Http404, HttpResponse, HttpResponsePermanentRedirect, HttpResponseRedirect,
 )
+from django.http.response import HttpResponseBase
 from django.template import loader
 from django.urls import NoReverseMatch, reverse
 from django.utils.functional import Promise
 
 
-def render(request, template_name, context=None, content_type=None, status=None, using=None):
+def render(request, template_name, context=None, content_type=None,
+           status=None, using=None, klass=HttpResponse):
```

Comment:
`response_class` instead of `klass`?

Reply:
Thanks, I am kind of indifferent, just used `klass` because I have seen it in other libraries.  Regardless, I will get around to making a ticket for this soon ...